wonjong/controllers/userController.js

```diff
@@ -0,0 +1,38 @@
+const userService = require('../services/userService');
+
+const signUp = async (req, res) => {
+  try {
+    const { name, email, password, profileImage } = req.body;
+    if (!name || !email || !password || !profileImage) {
+      return res.status(400).json({ message: 'KEY_ERROR' });
+    }
+    await userService.signUp(name, email, password, profileImage);
+    return res.status(201).json({
```

Comment:
코드 전체적으로 if문들 사이에 공백 추가해 주세요.
```suggestion
    const { name, email, password, profileImage } = req.body;
    
    if (!name || !email || !password || !profileImage) {
      return res.status(400).json({ message: 'KEY_ERROR' });
    }
    
    await userService.signUp(name, email, password, profileImage);
    return res.status(201).json({
```